server/auth/login.go

```diff
@@ -75,3 +88,69 @@ func randToken() string {
 	rand.Read(b)
 	return base64.StdEncoding.EncodeToString(b)
 }
+
+func SendNickName(message *messageService.Authentification)(string, error){
+	db, err := database.GetStorage()
+	if err != nil {
+		loger.Log.Errorf("Failed to open db andsend contacts", err)
+		return "", err
+	}
+	authentification := &messageService.Authentification{}
+	db.Where("user_name = ?", message.UserName).First(&authentification)
+	return authentification.NickName, nil
+}
+func SendContacts(cont *messageService.Authentification) (*messageService.Contacts, error) {
+
+	db, err := database.GetStorage()
+	if err != nil {
+		loger.Log.Errorf("Failed to open db andsend contacts", err)
+		return nil, err
+	}
+	var result []Result
+
+	rows, err := db.Table("Contacts").Select("main_user, contact_user, is_blocked").
+		Where("main_user = ? and is_blocked == 0", cont.UserName).Rows()
+	if err != nil {
+		loger.Log.Errorf("query from contacts error:", err)
+		return nil, err
+	}
+	var row Result
+	for rows.Next() {
+		if err := rows.Scan(&row.main_user, &row.contact_user, &row.is_blocked); err != nil {
+			loger.Log.Errorf("scan error:", err)
+			return nil, err
+		}
+		result = append(result, row)
+	}
+	var nick []string
+
+	for _, item := range result {
+		var nickRow NickName
+		rows, err := db.Table("Authentifications").Select("nick_name").
+			Where("user_name = ?", item.contact_user).Rows()
+		if err != nil {
+			loger.Log.Errorf("query from auth error:", err)
+			return nil, err
+		}
+		for rows.Next() {
+			if err := rows.Scan(&nickRow.nick_name); err != nil {
+				loger.Log.Errorf("scan error:", err)
+				return nil, err
+			}
+			nick = append(nick, nickRow.nick_name)
+		}
+	}
+
+	var co messageService.Contact
+	co = messageService.Contact{}
```

Comment:
we must set struct and then declare empty example of it for use in remote functions